server/tests/integration_test.rs

```diff
@@ -0,0 +1,998 @@
+
+    let mut line = String::new();
+
+    let mut output = BufReader::new(process.stdout.as_mut().unwrap());
+
+    // Ensure that the process has fully started.
+    while !line.contains("Starting Apache Arrow Flight") {
+        output
+            .read_line(&mut line)
+            .expect("Could not read line of process.");
+    }
+
+    return process;
+}
+
+/// Create a new Arrow Flight client to access the endpoints in the ModelarDB server.
+fn create_flight_service_client(
+    rt: &Runtime,
+    host: &str,
+    port: u16,
+) -> Result<FlightServiceClient<Channel>> {
+    let address = format!("grpc://{}:{}", host, port);
+
+    rt.block_on(async {
+        let fsc = FlightServiceClient::connect(address).await?;
+        Ok(fsc)
+    })
+}
+
+/// Create a model table in the ModelarDB server with Arrow Flight using the `do_action()` SQL Parser.
+fn create_model_table(
+    rt: &Runtime,
+    mut client: &mut FlightServiceClient<Channel>,
+    table_name: String,
+) -> std::result::Result<(), ()> {
+    let cmd = format!(
+        "CREATE MODEL TABLE {}(timestamp TIMESTAMP, value FIELD, tag TAG)",
+        table_name
+    );
+    let action = Action {
+        r#type: "CommandStatementUpdate".to_string(),
+        body: cmd.into_bytes(),
+    };
+
+    rt.block_on(async {
+        let mut _result = client.do_action(Request::new(action)).await;
+
+        Ok(())
+    })
+}
+
+/// Create a normal table without tags in the ModelarDB server with Arrow Flight using the `do_action()` SQL Parser.
+fn create_model_table_without_tags(
+    rt: &Runtime,
+    mut client: &mut FlightServiceClient<Channel>,
+    table_name: String,
+) -> std::result::Result<(), ()> {
+    let cmd = format!(
+        "CREATE MODEL TABLE {}(timestamp TIMESTAMP, value FIELD)",
+        table_name
+    );
+    let action = Action {
+        r#type: "CommandStatementUpdate".to_string(),
+        body: cmd.into_bytes(),
+    };
+
+    rt.block_on(async {
+        let mut _result = client.do_action(Request::new(action)).await;
+
+        Ok(())
+    })
+}
+
+/// Create a normal table in the ModelarDB server with Arrow Flight using the `do_action()` SQL Parser.
+fn create_table(
+    rt: &Runtime,
+    mut client: &mut FlightServiceClient<Channel>,
+    table_name: String,
+) -> std::result::Result<(), ()> {
+    let cmd = format!(
+        "CREATE TABLE {}(timestamp TIMESTAMP, values REAL, metadata REAL)",
+        table_name
+    );
+    let action = Action {
+        r#type: "CommandStatementUpdate".to_string(),
+        body: cmd.into_bytes(),
+    };
+
+    rt.block_on(async {
+        let mut _result = client.do_action(Request::new(action)).await;
+
+        Ok(())
+    })
+}
+
+/// Generate a [`RecordBatch`] with the current timestamp, a random value and an optional tag.
+fn generate_random_message(tag: Option<String>) -> (RecordBatch) {
+    let timestamp = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Can't generate the time.")
```

Comment:
Very minor but maybe change `Can't` to `Cannot`. However, it is purely a preference whether short forms are used in messages and comments.

Reply:
Fixed.